sycl/source/detail/scheduler/scheduler.cpp

```diff
@@ -72,6 +72,17 @@ EventImplPtr Scheduler::addCG(std::unique_ptr<detail::CG> CommandGroup,
   const bool IsKernel = CommandGroup->getType() == CG::KERNEL;
   const bool IsHostKernel = CommandGroup->getType() == CG::RUN_ON_HOST_INTEL;
   vector_class<StreamImplPtr> Streams;
+
+  if (IsKernel) {
+    Streams = ((CGExecKernel *)CommandGroup.get())->getStreams();
+    if (Queue->is_host()) {
```

Comment:
One should have a comment describing justification for such initialization of stream on host queue,

Reply:
Comment is added.